src/support_sphere/lib/main.dart

```diff
@@ -13,11 +16,27 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Support Sphere',
+      // App title
+      title: AppStrings.appName,
+      
+      // Theme configuration
       theme: _buildTheme(
         Brightness.light,
       ),
+
+      // Landing page
       home: const Landing(),
+
+      // Localizations configuration (i18n)
+      localizationsDelegates: const [
+        GlobalMaterialLocalizations.delegate,
+        GlobalWidgetsLocalizations.delegate,
+        GlobalCupertinoLocalizations.delegate,
+      ],
+      supportedLocales: const [
+        Locale('en'), // English
+        Locale('es'), // Spanish
```

Comment:
Should we comment this line out for now since we're only releasing the English localization to start?

Reply:
I'll leave it as spanish speakers are quite large around here 😄

Reply:
That's true, and reading the docs more it seems like components generated by flutter itself may have localization built in. I guess my concern comes from the fact that we're only providing strings/custom text in English at this point